frontend/src/components/Footer.js

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components/macro';
+
+export default function Footer() {
+    return (
+        <Wrapper>
+            <h5>powered by</h5>
```

Comment:
Do you have `h1` to `h4` on the page? Don't use headings for styling, better respect html semantics and use the correct level of heading and style it how you need it.